### DISCUSSION TITLE
how to cite this library?

### DISCUSSION THREAD

**Discussion by Abhijit-2592** [DIPlib/diplib#91] [General]
I am using this library for my research. Is there any preferred way to cite this library?

**Comment by crisluengo** [comment]
Thanks for the question!

With previous versions, just about all of our users used the MATLAB interface exclusively (DIPimage). We never expected citations, but quite a few people have cited its user manual. But that is probably not the right reference if you're using the new C++ library or its Python interface.

We should write a paper about the new library infrastructure, so we have something that can be referenced.

In the mean time, I guess you'll just have to reference the GitHub link.

**Reply by JanDenge** [reply]
Has this been update in the past years?

**Reply by crisluengo** [reply]
No, we still don’t have a paper.

we started writing one some years ago, but it wasn’t very interesting.

**Comment by Abhijit-2592** [comment]
Thanks I will cite the link then.
